drafts/developer-guide/pages/corteza-duplicate-detection/index.adoc

```diff
@@ -0,0 +1,85 @@
+:description: Learn how to USE {PRODUCT_NAME} duplicate detection to maintain clean and accurate data.
+
+{PRODUCT_NAME} through the Duplicate detection feature helps maintain clean and accurate data by using matching rules to apply a criterion that determines how closely a field on a new or edited record matches the same field on an existing record.  
+
+This is achieved through a customized user interface for handles duplicates: For example, block users from creating duplicates by displaying a warning and an error message.
+
+image::duplication-detection/duplicate-detection.gif[role="data-zoomable"]
+
+=== Features
+
+[Features and capabilities]
+|=======================
+|Feature |Description   | Level
+|General Detection  |Any field can be opted in for duplicate detection by a toggle option in field configurator. A general duplicate detection rule will be applied on such selected fields.     |Field
+|Detection Rule Definition |Multiple detection rules can be defined per Module, within compose module configurator    | Module
+|Detection on Single Record|Duplicate detection on Record Create / Update. A default/custom message will be shown based on configurations and will be allowed/restricted record save based on configuration|Record Page
+|Detection on All Records |Duplicate detection on all records of a selected module is an option available in all records page  to detect duplicates if any rules or fields are defined for duplicate detection.|All Records
+
+|=======================
+
+=== Benefits of corteza duplicate detection
+
+- To maintain data quality
+- For improved accuracy and efficiency in data processing and generating information
+- For effective description making
+- For optimizing data storage requirements
+- To minimize and manage errors
+
+=== Enabling Duplication Detection on a Module.
+
+Through validation, corteza ensures that no duplicate value is entered into the form for the designated field id when creating or editing a record.
+
+image::duplication-detection/validation.png[role="data-zoomable"]
+
+Validation is set up using two rules; Strict duplicate value validation and Soft duplicate value validation.
+
+* *Strict duplicate value validation*
+
+A record will not be saved when a duplicate value is detected in the selected fields in any existing record of this Module. 
+
+[NOTE]
+====
+When the Strict duplicate value validation rule is applied, an error message will be displayed on the screen's bottom right side, indicating that a record with the same name exists.
+====
+
+* *Soft duplicate value validation*
+
+This rule Record will not be saved when a duplicate value is detected in the selected fields in any existing record of this Module.
+
+[NOTE]
+====
+When Soft duplicate value validation is applied a warning message will be displayed on the bottom right side of the screen.
+==== 
+
+To enable the validation rules:
+
+1.Navigate to modules and create a new module.
```

Comment:
Can you not enable it for existing modules also?

Reply:
You can, I have covered that.